form/dynamic_form_modification.rst

```diff
@@ -490,9 +490,10 @@ sport like this::
                     $positions = null === $sport ? array() : $sport->getAvailablePositions();
 
                     $form->add('position', 'entity', array(
-                        'class'       => 'AppBundle:Position',
+                        'class' => 'AppBundle:Position',
```

Comment:
I agree. In #7907 we did that when using the `getRepository()` method. I will prepare another PR for other places where we use this (like here).